src/entity/Pickup.ts

```diff
@@ -658,12 +658,6 @@ export const pickups: IPickupSource[] = [
             );
           }
         }
-
-        // Now that the player unit's stamina has increased,sync the new
-        // stamina state with the player's predictionUnit so it is properly
-        // refelcted in the stamina bar
-        // (note: this would be auto corrected on the next mouse move anyway)
-        underworld.syncPlayerPredictionUnitOnly();
```

Comment:
The stamina bar still updates correctly without this?

Reply:
In my testing yes I think so. I couldn't make it not work.